app/controllers/api/integrations_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
Rubocop class length in this case should be fixed in a following PR instead of permanently hiding the issue with this comment

Reply:
because once we remove the `edit` completely, this will be easy to fix 😉

Reply:
I think I'm just going to disable my rubocop integration!

Reply:
hahahaah if that is the reason, you can have locally your own `.rubocop.yml` with your own rules. Ignored by `.git`